e2e/proxyIntegration.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { test } from '@playwright/test';
+import { clickPlaygroundRefreshButton } from './playground.spec';
+
+test.describe('Proxy integration', () => {
+  test('Proxy integration works on Playground, no network errors', async ({ page }) => {
+    // If any network request fails, fails the test
+    // This captures proxy integration failures that would otherwise go unnoticed thanks to default endpoint fallbacks
+    page.on('requestfailed', (request) => {
+      throw new Error(request.url() + ' ' + request?.failure()?.errorText);
+    });
```

Comment:
Have you tested if throwing error here will actually make the test fail? Because I'm having my suspicions that it will not 🤔

Reply:
I have, it will :) But I rewrote it to an `expect`, maybe that's more intuitive, good point. https://github.com/fingerprintjs/fingerprintjs-pro-use-cases/pull/116/commits/d30ac41fa6aad1df25b4c3f635c38dc90d36455f